lego/apps/achievements/tasks.py

```diff
@@ -0,0 +1,9 @@
+from lego import celery_app
+from lego.apps.achievements.promotion import check_all_promotions
+from lego.apps.events.tasks import AsyncRegister
+
+
+@celery_app.task(base=AsyncRegister, bind=True)
+def run_all_promotions(self, logger_context=None):
+    self.setup_logger(logger_context)
+    check_all_promotions()
```

Comment:
Maybe add a quick test for this too?

Reply:
Or I guess you kind of do that already, but with `check_all_promotions`. It annoys me that these lines are not covered though...